packages/devtools_app/lib/src/profiler/cpu_profile_model.dart

```diff
@@ -52,8 +51,8 @@ class CpuProfileData {
     // Each trace event in [subTraceEvents] will have the leaf stack frame id
     // for a cpu sample within [subTimeRange].
     final subTraceEvents = superProfile.stackTraceEvents
-        .where((trace) => subTimeRange
-            .contains(Duration(microseconds: trace[TraceEvent.timestampKey])))
+        .where((trace) =>
+            subTimeRange.contains(Duration(microseconds: trace['ts'])))
```

Comment:
why this change? seems like a minor regression to be using`'ts'` instead of `TraceEvent.timestampKey`
Optionally: maybe add an extension method for this?
e.g. 
trace.timeStamp

Reply:
I did this to remove the timeline_model import. `cpu_profile_model.dart` is used by both the html and flutter implementations of DevTools, and having it import one or the other was causing dependency issues. I'll add a TODO to use `TraceEvent.timestampKey` once we delete the html app.

Reply:
That is fine. you could probably also move TraceEvent into a separate library used in both places but it isn't critical. Probably makes sense to move TraceEvent outside of timeline_model at some point anyway as it is aa bit different than the rest of timeline_model.dart